matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/alias/DefaultAliasService.kt

```diff
@@ -26,9 +27,9 @@ internal class DefaultAliasService @AssistedInject constructor(
         private val addRoomAliasTask: AddRoomAliasTask
 ) : AliasService {
 
-    @AssistedInject.Factory
+    @AssistedFactory
     interface Factory {
-        fun create(roomId: String): AliasService
+        fun create(roomId: String): DefaultAliasService
```

Comment:
We are now returning the implementing type?

Reply:
Yes no choice, but it doesn't have any impact